cider/util/sqldefs.h

```diff
@@ -385,6 +392,8 @@ inline std::string toString(const SqlStringOpKind& kind) {
       return "REGEXP_REPLACE";
     case SqlStringOpKind::REGEXP_SUBSTR:
       return "REGEXP_SUBSTR";
+    case SqlStringOpKind::REGEXP_EXTRACT:
```

Comment:
please refactor this function to a hash map: SqlStringOpKind -> std::string

Reply:
updated